app/internal/event.py

```diff
@@ -1,15 +1,28 @@
+from collections import namedtuple
```

Comment:
You're not using this anymore, please remove.

Reply:
Actually, I still do (I think) :)